src/site/content/en/blog/prefers-reduced-motion/index.md

```diff
@@ -171,29 +171,67 @@ mediaQuery.addEventListener('change', () => {
 Note that the parentheses around the actual media query are obligatory:
 
 {% Compare 'worse' %}
+
 ```js
-window.matchMedia('prefers-reduced-motion: reduce')
+window.matchMedia('prefers-reduced-motion: reduce');
 ```
+
 {% endCompare %}
 
 {% Compare 'better' %}
+
 ```js
-window.matchMedia('(prefers-reduced-motion: reduce)')
+window.matchMedia('(prefers-reduced-motion: reduce)');
 ```
+
 {% endCompare %}
 
+## Working with the media query from `<picture>` contexts
+
+An interesting use case is to make playing of an animated AVIF, WebP, or GIF dependent on the
+`media` attribute. If `(prefers-reduced-motion: no-preference)` evaluates to `true`, it's safe to
+display the animated version, else the static version:
+
+```html
+<picture>
+  <!-- Animated versions. -->
+  <source
+    srcset="nyancat.avifs"
```

Comment:
You’ll want to add `type="image/avif"` (and similar below for the GIF) so the browser can select the first format it supports.

Reply:
Ah, right. It worked in practice, but explicitly adding the `type` is of course way better.

Reply:
Done https://github.com/GoogleChrome/web.dev/pull/8647/commits/20344f86d42e49b1441f5aca8d5164618cf45edc.